### ISSUE TITLE
No config file?

### ISSUE THREAD

**Issue by anechampenois** [ebruck/radiotray-ng#49]
As I wanted to modify the color of the icons (my desktop uses a white panel), I tried to find the config file according to the readme file. It was not here.
So I created it manually, but it doesn't seem to work. :(
FYI I used the tar.gz v0.2.0 and build my own Debian package (on Stretch ie Debian 9).

**Comment by ebruck** [comment]
What steps did you perform to build and install?

**Comment by anechampenois** [comment]
Thanks for your quick answer.

- To build and install, I 've used your readme: 
First I've installed the packages notified in the section 'To build on Ubuntu' (I suppose, as Ubuntu is a debian's derviative, it works too for Debian's building).
Then I followed the steps in the section 'Build Radiotray-NG & Debian Package'. It works like a charm and produced a .deb package for my system (i386). 

- Yes I've ran radiotray-ng first, and noticed the 'bug' of the icon color on my desktop. So I searched for a solution on this github, saw the config file modification. I've search it in my ~/.config/radiotray-ng directory  but it wasn't there. I've created it at this place with the name 'radiotray-ng.cfg'

- I changed the latest entries: 
"radiotray-ng-on" : "radiotray-ng-on" into "radiotray-ng-on" : "/usr/share/icons/ubuntu-mono-light/apps/24/radiotray-ng-on.svg"
and "radiotray-ng-off" : "radiotray-ng-off" into  "radiotray-ng-off" : "/usr/share/icons/ubuntu-mono-light/apps/24/radiotray-ng-off.svg"
It doesn't change anything on my system, even after rebooting it.

**Comment by ebruck** [comment]
The config file should be named:   radiotray-ng.json

**Comment by anechampenois** [comment]
Aha! So my mistake.
My confusion comes from the fact that there's a .cfg file in the same directory (ie ~/.config/radiotray-ng) for the bookmarks' editor.
So,

- I reverted the manual modifications i've made into radiotray-ng's files and deleted my false config file.

- I added the following entries in radiotray-ng.json:
"radiotray-ng-on" : "/usr/share/icons/ubuntu-mono-light/apps/24/radiotray-ng-on.svg" and  "radiotray-ng-off" : "/usr/share/icons/ubuntu-mono-light/apps/24/radiotray-ng-off.svg"
After enabling radiotray-ng, the icon in my tray is correctly changed. Thanks.
